package.json

```diff
@@ -123,6 +123,7 @@
     "eslint-plugin-notice": "~0.9.10",
     "eslint-plugin-playwright": "~0.16.0",
     "eslint-plugin-prettier": "~5.0.0",
+    "eslint-plugin-unicorn": "^48.0.1",
```

Comment:
**nitpick:** inconsistent declaration, should use `~` instead

Reply:
Good catch 😊